src/Scaffold/Console/model/model.stub

```diff
@@ -22,6 +22,28 @@ class {{studly_name}} extends Model
      */
     protected $fillable = [];
 
+    /**
+     * @var array Rules for fields
+     */
+    protected $rules = [];
+
+    /**
+     * @var array Cast fields
+     */
+    protected $casts = [];
+
+    /**
+     * @var array Json fields
+     */
+    protected $jsonable = [];
+
+    /**
+     * @var array Date fields
+     */
+    protected $date = [
+        'created_at',
+    ];
```

Comment:
The dates casting is done with `dates` not `date` and exists on the base model covering `created_at` and `updated_at` so it should not be encouraged to modify it unless actually necessary by adding more dates columns. If an argument can be made for including it in the stub then it should default to the defaults in the base model

Reply:
If i make it empty array it will still use predefined dates in Model?

Reply:
No, which means you'd have to either not include it or include it with the same defaults (i.e. just add `updated_at`)